python/foglamp/services/core/scheduler/scheduler.py

```diff
@@ -1024,6 +1077,168 @@ def _schedule_row_to_schedule(cls,
+                    task_id = key
+                    break
+            if task_id is None:
+                raise KeyError
+            task_process = self._task_processes[task_id]
+        except KeyError:
+            self._logger.info("No Task running for Schedule %s", schedule_id)
+            return False
+
+        schedule = task_process.schedule
+        if schedule.enabled is False:
+            self._logger.info("Schedule %s already disabled", schedule_id)
+            return True
+
+        # TODO: FOGL-356 track the last time TERM was sent to each task
+        task_process.cancel_requested = time.time()
+
+        # Terminate process
+        del self._schedules[schedule.id]
+        try:
+            # KNOWN ISSUE: Does not work well with microservices e.g. COAP etc
+            # TODO: If schedule is a microservice, then deal with shutdown, unregister etc.
+            task_process.process.terminate()
+            task_future = task_process.future
+            if schedule.type == Schedule.Type.STARTUP and task_future.cancel() is True:
+                await self._wait_for_task_completion(task_process)
+            self._logger.info(
+                "Disabled Schedule '%s/%s' process '%s' task %s pid %s\n%s",
+                schedule.name,
+                schedule.id,
+                schedule.process_name,
+                task_id,
+                task_process.process.pid,
+                self._process_scripts[schedule.process_name])
+        except ProcessLookupError:
+            pass  # Process has terminated
+
+        # Disable Schedule
+        # We need to recreate the _ScheduleRow for just one change - enabled = False. This is required as _ScheduleRow
+        # is of named tuple type which does not allow updation.
+        schedule_row = self._ScheduleRow(
+            id=schedule.id,
+            name=schedule.name,
+            type=schedule.type,
+            time=schedule.time,
+            day=schedule.day,
+            repeat=schedule.repeat,
+            repeat_seconds=schedule.repeat_seconds,
+            exclusive=schedule.exclusive,
+            enabled=False,
+            process_name=schedule.process_name)
+
+        self._schedules[schedule.id] = schedule_row
+
+        # Update database
+        update_payload = PayloadBuilder() \
+            .SET(enabled='f') \
+            .WHERE(['id', '=', str(schedule.id)]) \
+            .payload()
+        try:
+            self._logger.debug('Database command: %s', update_payload)
+            res = self._storage.update_tbl("schedules", update_payload)
+        except Exception:
+            self._logger.exception('Update failed: %s', update_payload)
+            raise RuntimeError('Update failed: %s', update_payload)
+        await asyncio.sleep(1)
+
+        return True
+
+    async def enable_schedule(self, schedule_id):
+        """
+        Get Schedule, Enable Schedule, Update database, Start Schedule
+
+        :param schedule_id:
+        :return:
```

Comment:
Please follow one style for docstring, as we have mentioned in **__template__.py**. Like below

Args:
Returns: